modules/react/src/components/export-video/export-video-panel-preview.js

```diff
@@ -91,7 +91,14 @@ export class ExportVideoPanelPreview extends Component {
     const map = this.mapRef.current.getMap();
     const deck = this.deckRef.current.deck;
     map.addLayer(new MapboxLayer({id: 'my-deck', deck}));
-    map.on('render', () => this.props.adapter.onAfterRender(() => this.forceUpdate()));
+    // TODO trying to make map scale to 100% of modal
+    // map.on('load', function () {
+    //   map.resize();
+    // });
+    map.on('render', () => this.props.adapter.onAfterRender(() => {this.forceUpdate()}));
+    map.resize();
```

Comment:
Did `map.resize();` fix an issue?

Reply:
Hmmm more of an alternative way of doing things I discovered while trying to get the preview to be 100% representative using Mapbox methods rather than ~25%. I found that I didn't need to pass in a resolution 2 components deep to the <DeckGL> component if I do resize() instead. However, that section is still in flux as I'm not sure if I'll be able to keep that code to get that coveted 100%.